pkg/backend/aio.go

```diff
@@ -65,7 +65,7 @@ func (s *Server) DeleteAioController(_ context.Context, in *pb.DeleteAioControll
 		if in.AllowMissing {
 			return &emptypb.Empty{}, nil
 		}
-		err := fmt.Errorf("unable to find key %s", in.Name)
+		err := status.Errorf(codes.NotFound, "unable to find key %s", in.Name)
```

Comment:
Similar to [this ](https://github.com/opiproject/opi-spdk-bridge/pull/305/files#r1151489439)comment.  Would it make sense to keep common errors defined in one place (maybe in server pkg) to avoid the duplication?

Reply:
good catch opened #306